src/System.ValueTuple/tests/TupleTests.cs

```diff
@@ -1,128 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This file was deleted because:
- It was only meant for netcoreapp, which we treat as a placeholder.
- It was using BinaryFormatter.